src/agoradatatools/gx.py

```diff
@@ -148,43 +150,61 @@ def convert_nested_columns_to_json(
             df[column] = df[column].apply(json.dumps)
         return df
 
-    def get_failed_expectations(self, checkpoint_result: CheckpointResult) -> str:
-        """Gets the failed expectations from a CheckpointResult and returns them as a formatted string
+    def set_warnings_and_failures(self, checkpoint_result: CheckpointResult) -> None:
+        """Sets class attributes for warnings and failures given a CheckpointResult
 
         Args:
             checkpoint_result (CheckpointResult): CheckpointResult object
-
-        Returns:
-            fail_message (str): String with information on which fields and expectations failed
```

Comment:
Is the standard to leave out the "Returns" comment if the function doesn't return anything? If not, please add one indicating that there's no return value. If so, maybe add that it doesn't return anything in the function comment.

Reply:
Oh I just noticed the `-> None` in the function declaration. So.. use your judgment on whether to comment or not.

Reply:
We have been leaving that section off in this codebase previously in these cases.